client/src/client.rs

```diff
@@ -1097,9 +1099,8 @@ impl Client {
     ///
     /// # Errors
     /// If sending request or decoding fails
-    pub fn set_config(&self, post_config: PostConfiguration) -> Result<bool> {
-        let body = serde_json::to_vec(&post_config)
-            .wrap_err(format!("Failed to serialize {post_config:?}"))?;
+    pub fn set_config(&self, dto: ConfigurationDTO) -> Result<bool> {
```

Comment:
Actually, `POST /configuration` doesn't return `true` anymore, so it shouldn't be `Result<bool>` here.